app/utils/shapes/heightAndWidthShape.ts

```diff
@@ -0,0 +1,95 @@
+import type TribontShape from 'mow-registry/models/tribont-shape';
+import {
+  DIMENSIONS,
+  dimensionToShapeDimension,
+  type shapeDimension,
+  type Shape,
+} from '.';
+import type Unit from 'mow-registry/models/unit';
+
+export default class HeightAndWidhtShape implements Shape {
+  height: shapeDimension;
+  width: shapeDimension;
+  shape: TribontShape;
+  constructor(
+    shape: TribontShape,
+    height: shapeDimension,
+    width: shapeDimension,
+  ) {
+    this.height = height;
+    this.width = width;
+    this.shape = shape;
+  }
+  toString() {
+    return `Hoogte: ${this.height.value} ${this.height.unit.symbol} Breedte: ${this.width.value} ${this.width.unit.symbol}`;
```

Comment:
no i18n?

Reply:
@lagartoverde ping, though I don't see any issues in the app, not sure where these strings are used? for debugging only?